package.json

```diff
@@ -33,16 +31,12 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "@discordjs/opus": "github:discordjs/opus",
     "apisauce": "^2.1.5",
-    "axios": "^0.21.1",
     "btbot-types": "^1.9.0",
     "discord-paginationembed": "^2.1.0",
     "discord.js": "^12.5.1",
     "dotenv": "^16.0.1",
```

Comment:
Serving the media stream from ytdl-core requires ffmpeg-static, unless we include it in the build even though its not explicity used